index.html

```diff
@@ -13424,193 +13424,346 @@ <h2>IANA Considerations</h2>
+          title="HTTP Request with profile requesting an expanded document">
+      <!--
+      GET /ordinary-json-document.json HTTP/1.1
+      Host: example.com
+      Accept: application/ld+json;****profile=http://www.w3.org/ns/json-ld#expanded****
+      -->
+      </pre>
+      <p>Requests the server to return the requested resource as JSON-LD
+        in <a>expanded document form</a>.</p>
 
-    <pre class="example" data-transform="updateExample"
-         data-content-type="http"
-         data-ignore
-         title="HTTP Request with profile requesting an expanded document">
-    <!--
-    GET /ordinary-json-document.json HTTP/1.1
-    Host: example.com
-    Accept: application/ld+json;****profile=http://www.w3.org/ns/json-ld#expanded****
-    -->
-    </pre>
-    <p>Requests the server to return the requested resource as JSON-LD
-      in <a>expanded document form</a>.</p>
+      <pre class="example" data-transform="updateExample"
+          data-content-type="http"
+          data-ignore
+          title="HTTP Request with profile requesting a compacted document">
+      <!--
+      GET /ordinary-json-document.json HTTP/1.1
+      Host: example.com
+      Accept: application/ld+json;****profile=http://www.w3.org/ns/json-ld#compacted****
+      -->
+      </pre>
+      <p>Requests the server to return the requested resource as JSON-LD
+        in <a>compacted document form</a>.
+        As no explicit context resource is specified, the server compacts
+        using an application-specific default context.</p>
 
-    <pre class="example" data-transform="updateExample"
-         data-content-type="http"
-         data-ignore
-         title="HTTP Request with profile requesting a compacted document">
-    <!--
-    GET /ordinary-json-document.json HTTP/1.1
-    Host: example.com
-    Accept: application/ld+json;****profile=http://www.w3.org/ns/json-ld#compacted****
-    -->
-    </pre>
-    <p>Requests the server to return the requested resource as JSON-LD
-      in <a>compacted document form</a>.
-      As no explicit context resource is specified, the server compacts
-      using an application-specific default context.</p>
+      <pre class="example" data-transform="updateExample"
+          data-content-type="http"
+          data-ignore
+          title="HTTP Request with profile requesting a compacted document with a reference to a compaction context">
+      <!--
+      GET /ordinary-json-document.json HTTP/1.1
+      Host: example.com
+      Accept: application/ld+json;****profile="http://www.w3.org/ns/json-ld#flattened http://www.w3.org/ns/json-ld#compacted"****
+      -->
+      </pre>
+      <p>Requests the server to return the requested resource as JSON-LD
+        in both <a>compacted document form</a>
+        and <a>flattened document form</a>.
+        Note that as whitespace is used to separate the two URIs, they
+        are enclosed in double quotes (<code>"</code>).</p>
+    </section>
+  </section>
+  <section>
```

Comment:
Predictable section identifier.

```suggestion
  <section id="structured-extension-ld-json">
```